app/assets/stylesheets/modules/sul-footer.scss

```diff
@@ -0,0 +1,55 @@
+#su-content {
+  padding-bottom: 200px !important;
+}
+
+#sul-footer-container {
+  padding: 0 !important;
+  position: relative;
+  margin-top: -200px;
+  min-height: 200px;
+  clear: both;
+  -webkit-box-shadow: 0 4px 8px -8px rgba(0,0,0,0.2) inset;
+  box-shadow: 0 4px 8px -8px rgba(0,0,0,0.2) inset;
+  background-color: $sul-footer-bg-color;
+  }
+
+#sul-footer {
+  display: table;
+  height: 80px;
+  padding: 0px;
+}
+
+#sul-footer-img {
+  display: table-cell;
+  vertical-align: middle;
+  width: 162px;
+}
+
+#sul-footer-links {
+  display: table-cell;
+  vertical-align: middle;
+  text-align: left;
+  padding-left: 15px;
+  font-size: small;
+}
+
+#sul-footer li a {
+  color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.